conda_build/jinja_context.py

```diff
@@ -167,6 +167,43 @@ def load_npm():
         return json.load(pkg)
 
 
+def load_file_regex(config, load_file=None, regex_pattern=None, from_recipe_dir=False,
+                    recipe_dir=None, permit_undefined_jinja=True):
+    import re
+    match = False
+
+    cd_to_work = False
+
+    if load_file is None:
+        raise RuntimeError('File to be searched is not specified.')
+
+    if from_recipe_dir and recipe_dir:
+        load_file = os.path.abspath(os.path.join(recipe_dir, load_file))
+    elif os.path.exists(config.work_dir):
```

Comment:
I should add as well that the default execution path works well for the actual use case I have in #1316, so I know it works in at least one case, its just a matter of writing the test for it.